BaSD-2021/Dami-MindSet-server
# MS-55: Applications - CRUD

Tasks:

HTML & CSS
- Create a html layout with Applications's data in a table
- Create a modal for each crud operation with a form 

JS:
- Create a post request to create an application
- Create a get request to read all application
- Create a get request to read one application
- Create a put request to update an interview
- Create a delete request to remove an interview

Validations:
- Validate each field in the modal form 

TO DO: 
Tasks:

HTML & CSS
- Create a html layout with Inteview's data in a table
- Create a modal for each crud operation with a form 

JS:
- Create a post request to create an interview
- Create a get request to read all interviews
- Create a get request to read one interview
- Create a put request to update an interview
- Create a delete request to remove an interview

Validations:
- Validate each field in the modal form 

TO DO: 
- Create a get request & a modal to read one interview
- Fix update put request --> Problems with fetch
- Fix remove confirm modal --> When an application is remove, the modal shows the fields with undefined
-  Use envoirment variables or window.location.origin for all the URLs
